src/WasmSdk/Tasks/FileHasher.cs

```diff
@@ -6,7 +6,7 @@
 using System.Security.Cryptography;
 using System.Text;
 
-namespace Microsoft.NET.Sdk.BlazorWebAssembly
+namespace Microsoft.NET.Sdk.WebAssembly
 {
     public static class FileHasher
```

Comment:
@MackinnonBuck this will move to static web assets as part of https://github.com/dotnet/aspnetcore/issues/47000